configs/config.go

```diff
@@ -0,0 +1,13 @@
+package configs
+
+type Configs struct {
+	Port string `json:"port"`
```

Comment:
Did you format your code with `goimports` or `gofmt`? All struct comments should be aligned the same way.